cla_backend/apps/guidance/management/commands/load_markdown_files.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import markdown
+import os
+import re
+import sys
+from django.core.management.base import BaseCommand
+
+from guidance.models import Note, Tag, NoteTagRelation
+
+meta_regex = re.compile('---\n(.*?)---\n', re.DOTALL)
+title_regex = re.compile('title: (.*?)\n')
+date_regex = re.compile('date: "(.*?)"\n')
+tag_regex = re.compile('  - (.*?)\n')
+
+
+class Command(BaseCommand):
+
+    help = 'Load Notes in from old markdown files'
+    requires_model_validation = True
+
+    def handle(self, *args, **options):
+
+        if len(args) != 1:
+            self.stdout.write("Must pass directory for markdown files as arg")
+            sys.exit(-1)
```

Comment:
Why -1? Exit codes normally must be between 0 and 127, with 0 being ok, 1 for an error and sometimes 2 for a syntax error.